managed/services/telemetry/config.go

```diff
@@ -28,17 +29,25 @@ import (
 	"github.com/percona/pmm/managed/utils/envvars"
 )
 
+const (
+	envDisableSend           = "PERCONA_TEST_TELEMETRY_DISABLE_SEND"
+	envConfigFile            = "PERCONA_TEST_TELEMETRY_FILE"
+	envDisableStartDelay     = "PERCONA_TEST_TELEMETRY_DISABLE_START_DELAY"
+	envReportingInterval     = "PERCONA_TEST_TELEMETRY_INTERVAL"
+	envReportingRetryBackoff = "PERCONA_TEST_TELEMETRY_RETRY_BACKOFF"
```

Comment:
Please add these variables to this table: https://github.com/percona/pmm/blob/main/managed/CONTRIBUTING.md#advanced-setup

Reply:
done